pipelines/imputation/testing/InputQcEmpty.wdl

```diff
@@ -0,0 +1,50 @@
+version 1.0
+
+workflow InputQC {
+    String pipeline_version = "0.0.1"
+
+    input {
+        Int chunkLength = 25000000
```

Comment:
purely for my own curiosity and understanding: how do we determine what a good value is for chunkLength?

Reply:
we set this in consultation with Brian Browning and our scientific advisors. I don't recall the exact rationale for this particular number. (of course here the value doesn't matter, and it's not checked in the real InputQc wdl either.)